helm/external-dns-app/templates/deployment.yaml

```diff
@@ -67,6 +67,9 @@ spec:
         {{- range .Values.externalDNS.sources }}
         - --source={{ . }}
         {{- end }}
+        {{- if eq .Values.provider "vmware" }}
```

Comment:
Can you add a note in the `values.yaml` file that the `externalDNS.sources` value is ignored if `provider == 'vmware'` ?

Reply:
ah no it won't get ignored, it just adds an additional arg to support vmware. `--sources` can be provided as many times as you like, so by default on vmware the chart will have:

```
--source=service
--source=crd
```

Reply:
ah yes, thats good :)